docs/source/import_export/main.rst

```diff
@@ -9,30 +9,29 @@ Export
 
 Use ``verdi export create`` to export a selection of nodes, codes, groups, users, or computers.
 
-* **Selection:** Nodes can be selected via identifier (PK, UUID, label) or by exporting predefined groups of nodes.
-* **Augmentation:** By default, the export function augments the selection of nodes by their parents in order to preserve the provenance.
-  For calculation nodes, the direct outputs are also added.
-  By default, comments and logs are also added for each exported node.
-  These may be excluded using the proper flags.
-  There are several other options to export specific parts of the provenance graph, starting from the selection.
-  For more information about this please read the :ref:`corresponding subsection<consistency>` of the :ref:`Provenance section<concepts_provenance>`.
+See ``verdi export create -h`` for a full list of available options.
+
+* **Selection:** Nodes can be selected via identifier (PK, UUID, label).
+  It is often convenient to first put the nodes you would like to export into a :py:class:`~aiida.orm.Group` and then to export the group using ``verdi export create -G <group-name>``.
```

Comment:
I am not sure this will work as a link to the Group class? Unless it has been fixed, the link path should be absolute, i.e.:
```suggestion
  It is often convenient to first put the nodes you would like to export into a :py:class:`~aiida.orm.groups.Group` and then to export the group using ``verdi export create -G <group-name>``.
```

Reply:
I just checked and it works :) I would also consider it a sphinx bug, if it did not issue a warning if I provided a path it does not understand.